test/e2e/specs/editor-tracking/editor-tracking__fse-template-events.ts

```diff
@@ -216,14 +216,7 @@ describe(
 
 			it( '"wpcom_block_editor_template_part_detach_blocks" event fires with correct "block_names" and "template_part_id"', async function () {
 				const eventDidFire = await editorTracksEventManager.didEventFire(
-					'wpcom_block_editor_template_part_detach_blocks',
-					{
```

Comment:
**Question**: Can you tell me a little more about removing this property check? From what I see testing this, it looks like it still does fire the event with these properties, and the tests should pass with this left in.

Reply:
So this is a weird one. I agree the event was firing with correct meta. I had the tracks tool open while testing that. Despite all that, it was failing here. I has assume it would be ok to just test the event without meta, so I pushed that rather than investigating further. 

I reverted the change to start evaluating in response to your comment here - and it works without issue. Not sure if something changed in my environment or it was some user error on my part or what, but a test that had been failing for me last week seems to be working without need for any changes now. I've simply reverted the change.

Reply:
>Not sure if something changed in my environment or it was some user error on my part or what, but a test that had been failing for me last week seems to be working without need for any changes now.

Hurray, the joys of E2E testing!! 😒 🤣 Argh, that's so frustrating! Something very well may have been thrown off in the environment at that time. This seems like a safe revert for now though, thank you! 😄